build/Android.props

```diff
@@ -0,0 +1,18 @@
+﻿<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <PackageReference Include="System.Collections" Version="4.3.0" />
```

Comment:
The VS tooling added those dependencies so I have kept them, but they are not necessary as it seems. Removed in PR.